smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/commit/KafkaThrottledLatestProcessedCommit.java

```diff
@@ -82,10 +73,16 @@ public static KafkaThrottledLatestProcessedCommit create(KafkaConsumer<?, ?> con
             Map<String, String> config,
             KafkaSource<?, ?> source) {
 
-        int maxPollRecords = Integer.parseInt(config.getOrDefault(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, "500"));
-        int maxReceivedWithoutAckAllowed = getNextPowerOfTwoEqualOrGreater(maxPollRecords * 2);
-        log.settingMaxReceivedWithoutAckAllowed(config.get(ConsumerConfig.GROUP_ID_CONFIG), maxReceivedWithoutAckAllowed);
-        return new KafkaThrottledLatestProcessedCommit(consumer, source, maxReceivedWithoutAckAllowed);
+        int unprocessedRecordMaxAge = Integer.parseInt(config.getOrDefault("throttled.unprocessed-record-max-age", "60000"));
```

Comment:
you should store the IncomingKAfkaConfiguration and use this object instead of accessing the config. Typically, alias and default values are managed for you.

Reply:
I see it. thanks.